js/bybit.js

```diff
@@ -4877,27 +4885,127 @@ module.exports = class bybit extends Exchange {
         //        "time_now": "1645085118.727358"
         //    }
         //
+        // USDC Settled
+        //
+        //     {
+        //         "retCode": 0,
+        //         "retMsg": "",
+        //         "result": [
+        //             {
+        //                 "symbol": "BTCPERP",
+        //                 "timestamp": "1664406000",
+        //                 "openInterest": "44142800000"
+        //             },
+        //         ]
+        //     }
+        //
         const result = this.safeValue (response, 'result');
         return this.parseOpenInterests (result, market, since, limit);
     }
 
+    async fetchOpenInterest (symbol, params = {}) {
+        /**
+         * @method
+         * @name bybit#fetchOpenInterest
+         * @description Retrieves the open interest of a currency
+         * @see https://bybit-exchange.github.io/docs/futuresV2/inverse/#t-marketopeninterest
+         * @see https://bybit-exchange.github.io/docs/usdc/perpetual/#t-queryopeninterest
+         * @param {string} symbol Unified CCXT market symbol
+         * @param {object} params exchange specific parameters
+         * @param {string|undefined} params.timeframe "5m", 15m, 30m, 1h, 4h, 1d
+         * @param {int|undefined} params.limit The number of open interest structures to return. Max 200, default 50
+         * @returns {object} an open interest structure{@link https://docs.ccxt.com/en/latest/manual.html#interest-history-structure}
+         */
+        const timeframe = this.safeString (params, 'timeframe', '1h');
```

Comment:
Should `timeframe` just be a parameter? Have you written this method for a few more exchanges and found out what the common parameters would be?

Reply:
Bybit is the only exchange I've seen use `timeframe` so far, usually they only require a market id and return the current open interest